views/layout.html

```diff
@@ -9,31 +9,15 @@
       <link rel="stylesheet" href="css/font_awesome.css">
       <link class='active-theme' rel="stylesheet" href="css/themes/empty.css">
       <script>
-      var require = {
-         paths: {
-            "jquery":"//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min",
-            "cookies":"//cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min",
-            "bootstrap":"//netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min",
-            "socketjs":"//cdnjs.cloudflare.com/ajax/libs/socket.io/<%= socketVersion %>/socket.io.min",
-            "underscore":"//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.6.0/underscore-min",
-            "text":"/lib/requirejs-text/text",
-            "html":"/html",
-            "spec":"/spec"
-         },
-         shim: {
-            'bootstrap': ['jquery'],
-            'socketio': {
-               exports: 'io'
-            },
-            'debug': {
-               exports: 'debug'
-            }
-         }
-      };
+         var App = {
+            socketToken: "<%= socketToken %>",
+            user: "<%= user %>",
+            debug: <%= debug %>
```

Comment:
Does this introduce any xss vulnerabilities?

Reply:
Never mind, this is moved from another file.

Reply:
It doesn't _introduce_ any ;)

I haven't tried though. Maybe it's better to get this information as JSON from the backend instead of templating it. I'm okay with adding another issue to look into changing how App is populated.